ajax/libs/moonjs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "moonjs",
+  "filename": "moon.min.js",
+  "version": "0.10.0",
+  "description": "A minimal, blazing fast UI library.",
+  "homepage": "http://moonjs.ga/",
+  "keywords": [
+    "UI",
+    "MVC",
+    "Library",
+    "JS"
```

Comment:
We can use the keywords from upstream. Ref: https://github.com/KingPixil/moon/blob/v0.10.0/package.json#L16-L19